crates/biome_js_analyze/src/lint/nursery/no_restricted_imports.rs

```diff
@@ -36,14 +41,24 @@ declare_rule! {
     }
 }
 
+/// Options for each path of rule `noRestrictedImports`.
+#[derive(Clone, Debug, Default, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct RestrictedModuleConfig {
```

Comment:
Can you please document each new option that you created? The doc documents well with used for the JSON schema, and we want to document everything that is public.